boards/espressif/esp32s3_devkitc/esp32s3_devkitc_procpu.dts

```diff
@@ -29,11 +31,6 @@
 		zephyr,bt-hci = &esp32_bt_hci;
 	};
 
-	aliases {
-		uart-0 = &uart0;
-		sw0 = &button0;
-	};
-
```

Comment:
These changes have nothing to do with "Add default zephyr_udc0 label for USB-OTG.". You may consider them harmless, but that is not how the commit should be. Please rework it and create a separate commit for the aliases property changes.